.ci/.docker-images.yml

```diff
@@ -0,0 +1,389 @@
+
+## This is also failing in Jenkins
+## https://apm-ci.elastic.co/job/apm-shared/job/docker-images/job/weblogic/
+#  - name: "weblogic"
+#    prepare_script: docker pull store/oracle/weblogic:12.2.1.3-dev
+#    build_script: docker tag store/oracle/weblogic:12.2.1.3-dev ${REGISTRY}/${PREFIX}/weblogic:12.2.1.3-dev
+#    push_script: docker push ${REGISTRY}/${PREFIX}/weblogic:12.2.1.3-dev
+
+  # Opbeans Docker Images
+
+  - <<: *opbeans-docker-images
+    name: "opbeans-dotnet"
+    repository: "elastic/opbeans-dotnet"
+
+  - <<: *opbeans-docker-images
+    name: "opbeans-node"
+    repository: "elastic/opbeans-node"
+
+  - <<: *opbeans-docker-images
+    name: "opbeans-python"
+    repository: "elastic/opbeans-python"
+
+  - <<: *opbeans-docker-images
+    name: "opbeans-frontend"
+    repository: "elastic/opbeans-frontend"
+
+  - <<: *opbeans-docker-images
+    name: "opbeans-java"
+    repository: "elastic/opbeans-java"
+
+  - <<: *opbeans-docker-images
+    name: "opbeans-go"
+    repository: "elastic/opbeans-go"
+
+  - <<: *opbeans-docker-images
+    name: "opbeans-loadgen"
+    repository: "elastic/opbeans-loadgen"
+
+  - <<: *opbeans-docker-images
+    name: "opbeans-php"
+    repository: "elastic/opbeans-php"
+
+  - <<: *opbeans-docker-images
+    name: "opbeans-ruby"
+    repository: "elastic/opbeans-ruby"
+
+  # APM Pipeline Library Docker Images
+
+  - <<: *apm-agent-library-docker-images
+    name: "apache-ant"
+    test_script: "cd .. && make test-apache-ant"
+    working_directory: ".ci/docker/apache-ant"
+
+  - <<: *apm-agent-library-docker-images
+    name: "dind-buildx"
+    test_script: "cd .. && make test-dind-buildx"
+    working_directory: ".ci/docker/dind-buildx"
+
+  - <<: *apm-agent-library-docker-images
+    name: "github-label-sync"
+    test_script: "cd .. && make test-github-label-sync"
+    working_directory: ".ci/docker/github-label-sync"
+
+  - <<: *apm-agent-library-docker-images
+    name: "gren"
+    test_script: "cd .. && make test-gren"
+    working_directory: ".ci/docker/gren"
+
+  - <<: *apm-agent-library-docker-images
+    name: "shellcheck"
+    test_script: "cd .. && make test-shellcheck"
+    working_directory: ".ci/docker/shellcheck"
+
+  - <<: *apm-agent-library-docker-images
+    name: "yamllint"
+    test_script: "cd .. && make test-yamllint"
+    working_directory: ".ci/docker/yamllint"
+
+## This is also failing in Jenkins
+## https://apm-ci.elastic.co/job/apm-shared/job/docker-images/job/kibana-devmode/
+#  - <<: *apm-agent-library-docker-images
+#    name: "kibana-yarn"
+#    test_script: "cd .. && make test-kibana-yarn"
+#    working_directory: ".ci/docker/kibana-yarn"
+
+## This is also failing in Jenkins
+## https://apm-ci.elastic.co/job/apm-shared/job/docker-images/job/kibana-devmode/
+#  - <<: *apm-agent-library-docker-images
+#    name: "kibana-devmode"
+#    test_script: "cd .. && make test-kibana-devmode"
+#    working_directory: ".ci/docker/kibana-devmode"
```

Comment:
I have to take a look. These are the base images to build Kibana in the deploy Kibana PR, it should be some change in the build system that breaks the build

Reply:
I also checked these images.

- https://github.com/search?q=org%3Aelastic+kibana-devmode&type=code
- https://github.com/search?q=org%3Aelastic+kibana-yarn&type=code

The only thing I could find is that `kibana-yarn` is used by `kibana-devmode`, but `kibana-devmode` is not used anywhere.

Reply:
yep, `kibana-yarn` is the only used

Reply:
Will be handled in https://github.com/elastic/apm-pipeline-library/issues/1992